### ISSUE TITLE
Is it still maintained?

### ISSUE THREAD

**Issue by rubyFeedback** [SpellholdStudios/Improved_Asylum#2]
Is it still maintained?

**Comment by skellytz** [comment]
Yes, SHS Staff will take over the maintenance of this project in its [new repository](https://github.com/Spellhold-Studios/Improved-Asylum).
